src/mscorlib/corefx/System/Globalization/CompareInfo.cs

```diff
@@ -37,6 +37,10 @@ public enum CompareOptions
         OrdinalIgnoreCase = 0x10000000,   // This flag can not be used with other flags.
         StringSort = 0x20000000,   // use string sort method
         Ordinal = 0x40000000,   // This flag can not be used with other flags.
+        
+        // UpperCaseFirst = 0x01000000
```

Comment:
We already have API's that throw PNSE() ... would it make sense to expose this API and have anyone that use it throw PNSE() when on Windows? That seems more consistent.

I don't feel super strongly, if you would rather keep it this way and open an issue to support it somehow on Windows later.